### PR TITLE
[FIX] Issues

### PR DESCRIPTION
**Pull request by oliveralka**
add column header
explain manual validation
remove copy spectral lib

Fixes issues:
#115 
#116 
#117

### CODE REVIEW

**Comment by oliveralka** [comment]
mark digression (smiles, RDKit (not in metabolomics))
highlight advances params
explain light gray color params

further fixes:
#108 
#109 
#110

File: Handout/metabolomics.tex [hunk 1]
```diff
@@ -283,20 +285,13 @@ \subsubsection{Spectral library search}
...
-	\item macOS \newline \directory{ Applications/ KNIME 3.3.app / Contents / Eclipse / plugins / \newline 
-de.openms.macosx.x86\_64\_2.1.0.201611191105 / payload / share / OpenMS / CHEMISTRY}
-\end{itemize}
-Now construct the workflow as shown in Fig.~\ref{fig:speclib}.
-Use the file \directory{Example\_Data / Metabolomics / datasets \newline / Metabolite\_ID\_SpectraDB\_positive.mzML} as input for your workflow. It contains tandem spectra that are identified by the \KNIMENODE{MetaboliteSpectralMatcher}. The resulting mzTab file is read back into a KNIME table and stored in an Excel table. Make sure that you connect the MzTabReader port corresponding to the Small Molecule Section to the \KNIMENODE{Excel writer (XLS)}).
+Construct the workflow as shown in Fig.~\ref{fig:speclib}.
+Use the file \directory{Example\_Data / Metabolomics / datasets \newline / Metabolite\_ID\_SpectraDB\_positive.mzML} as input for your workflow.  You can use the spectral library from \\
+\directory{Example\_Data / Metabolomics / databases / MetaboliteSpectralDB.mzML}\\ on the USB stick as second input.
```
Comment (timosachsenberg) [inline Handout/metabolomics.tex]: doesn't need to be on USB stick

File: Handout/metabolomics.tex [hunk 2]
```diff
@@ -333,7 +328,7 @@ \subsubsection{Manual validation}
 \end{figure}
 
 \begin{task}
-On the METLIN homepage search for \menu{Compound Name} Glutathione using the \menu{Advanced Search} (\url{https://metlin.scripps.edu/landing_page.php?pgcontent=advanced_search}). Note that free registration is required. Which collision energy (and polarity) gives the best (visual) match to your experimental spectrum in TOPPView?
+On the METLIN homepage search for \menu{Compound Name} Glutathione using the \menu{Advanced Search} (\url{https://metlin.scripps.edu/landing_page.php?pgcontent=advanced_search}). Note that free registration is required. Which collision energy (and polarity) gives the best (visual) match to your experimental spectrum in TOPPView?Here you can compare the fragmentation patterns in both spectra shown by the Intensity or relative Intensity, the mz of a peak and the distance between peaks. Each distance between two peaks corresponds to a fragment of elemental composition (e.g. NH2 with the charge of one would have mass of two peaks of 16.023 Th).
```
Comment (timosachsenberg) [inline Handout/metabolomics.tex]: space after TOPPView?
mz -> m/z
(e.g. -> (e.g.,

**Comment by timosachsenberg** [comment]
thanks!

**Pull request merged**